staging/src/k8s.io/component-base/logs/json/register/register.go

```diff
@@ -17,13 +17,16 @@ limitations under the License.
 package register
 
 import (
+	"k8s.io/apimachinery/pkg/util/version"
 	logsapi "k8s.io/component-base/logs/api/v1"
 	json "k8s.io/component-base/logs/json"
 )
 
 func init() {
 	// JSON format is optional klog format
-	if err := logsapi.RegisterLogFormat(logsapi.JSONLogFormat, json.Factory{}, logsapi.LoggingBetaOptions); err != nil {
+	if err := logsapi.RegisterLogFormat(logsapi.JSONLogFormat, json.Factory{}, logsapi.LoggingStableOptions,
+		logsapi.VersionedFeature{Version: version.MajorMinor(1, 23), Feature: logsapi.LoggingBetaOptions},
```

Comment:
```suggestion
		// TODO: when promoting JSON to GA, keep this entry for
		// three more releases because of version emulation,
		// then remove it.
		logsapi.VersionedFeature{Version: version.MajorMinor(1, 23), Feature: logsapi.LoggingBetaOptions},
```

Is "three" releases correct?

Reply:
Yes, that's correct